ultimate_cron.install

```diff
@@ -209,7 +206,7 @@ function ultimate_cron_requirements($phase) {
         );
         $requirements['description'][] = t(
           'Some jobs are behind their schedule. Please check if !link is running properly.', array(
-          '!link' => l(t('cron'), 'admin/config/system/cron')
+          '!link' => 'admin/config/system/cron',
```

Comment:
not correct, this needs to go through Url::fromRoute() or it won't work propery (also means we're missing tests). And the link right now points to the wrong page.